src/viam/robot/client.py

```diff
@@ -710,7 +712,24 @@ async def transform_pose(
 
         ::
 
-            pose = await machine.transform_pose(PoseInFrame(), "origin")
+            from viam.proto.common import Pose, PoseInFrame
+
+            pose = Pose(
+                x=1.0,    # X coordinate in mm
+                y=2.0,    # Y coordinate in mm
+                z=3.0,    # Z coordinate in mm
+                o_x=0.0,  # Orientation radian X
+                o_y=0.0,  # Orientation radian Y
+                o_z=0.0,  # Orientation radian Z
+                theta=0.0 # Orientation angle in radians
+            )
+
+            pose_in_frame = PoseInFrame(
+                reference_frame="world",
+                pose=pose
+            )
+
+            pose = await machine.transform_pose(pose_in_frame, "world")
```

Comment:
should this overwrite the pose? Or should the variable be called something else?